lib/builtin.pm

```diff
@@ -106,6 +106,14 @@ The following bundles currently exist:
     :5.40      true false weaken unweaken is_weak blessed refaddr reftype
                ceil floor is_tainted trim indexed
 
+=head2 Function Optimisations
+
+Various optimisations that apply to many functions in the L<builtin> package
+would be broken if the functions are ever replaced or changed, such as by
```

Comment:
I assume this means "broken" as in "unable to be applied", not "breaking your program"? That could be made clearer.

Reply:
Ahyes, fair. Adjusted wording.